myhpi/core/context.py

```diff
@@ -39,6 +40,7 @@ def base_context(request):
         "root_page": root_page,
         "pages_by_parent": page_lookup,
         "minutes_creation_links": minutes_creation_links,
+        "enable_menu_cache": not settings.DEBUG,
```

Comment:
you could also set the template_cache_duration here and use that in the template to avoid duplication and use a value of 0 or 1 seconds in debug mode

Reply:
you're right, did it now. Thinking about that: It could also just be a setting/.env key where the .env.example contains 1 and the production .env has a different value (e.g. 500)

Reply:
Does the thumbs up mean you want it as a settings key in this pr or not?

Reply:
If you want to include it in this PR go ahead, but I don't think it is necessary as we currently don't have any other usecase apart from the one in this PR